solo-chains/runtime/dancelight/src/bridge_to_ethereum_config.rs

```diff
@@ -17,11 +17,47 @@
 //! The bridge to ethereum config
 
 pub const SLOTS_PER_EPOCH: u32 = snowbridge_pallet_ethereum_client::config::SLOTS_PER_EPOCH as u32;
+
 use {
-    crate::{parameter_types, weights, Runtime, RuntimeEvent},
+    crate::{
+        parameter_types, weights, AggregateMessageOrigin, Balance, Balances, EthereumOutboundQueue,
+        EthereumSystem, FixedU128, GetAggregateMessageOrigin, Keccak256, MessageQueue, Runtime,
+        RuntimeEvent, TreasuryAccount, WeightToFee, UNITS,
+    },
+    pallet_xcm::EnsureXcm,
     snowbridge_beacon_primitives::{Fork, ForkVersions},
+    snowbridge_core::{gwei, meth, AllowSiblingsOnly, PricingParameters, Rewards},
+    sp_core::{ConstU32, ConstU8},
 };
 
+parameter_types! {
+    pub Parameters: PricingParameters<u128> = PricingParameters {
+        exchange_rate: FixedU128::from_rational(1, 400),
+        fee_per_gas: gwei(20),
+        rewards: Rewards { local: 1 * UNITS, remote: meth(1) },
+        multiplier: FixedU128::from_rational(1, 1),
+    };
+}
+
+// https://github.com/paritytech/polkadot-sdk/blob/2ae79be8e028a995b850621ee55f46c041eceefe/cumulus/parachains/runtimes/bridge-hubs/bridge-hub-westend/src/bridge_to_ethereum_config.rs#L105
+impl snowbridge_pallet_outbound_queue::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type Hashing = Keccak256;
+    type AggregateMessageOrigin = AggregateMessageOrigin;
+    type GetAggregateMessageOrigin = GetAggregateMessageOrigin;
+    type MessageQueue = MessageQueue;
+    type Decimals = ConstU8<12>;
+    type MaxMessagePayloadSize = ConstU32<2048>;
```

Comment:
it seems to be this will be not enough, maybe we should double it.

in the case of 100 validators, 32 byte per validator account * 100 is already 3200...

Reply:
We decided not to do this and instead send the message in batches, correct?

Reply:
not in batches, but to send only the root. so yeah keep it like this

Reply:
ignore